frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSdkSettings/CorsOriginInput.tsx

```diff
@@ -0,0 +1,58 @@
+import { jt, t } from "ttag";
+
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { PLUGIN_EMBEDDING_SDK } from "metabase/plugins";
+import { Box } from "metabase/ui";
+
+import SettingHeader from "../../SettingHeader";
+import { SetByEnvVarWrapper } from "../../SettingsSetting";
+import { SettingTextInput } from "../../widgets/SettingTextInput";
+import { useEmbeddingSetting, useMergeSetting } from "../hooks";
+
+import { useEmbeddingSettingsLinks } from "./sdk";
+
+export const CorsOriginInput = () => {
+  const isEE = PLUGIN_EMBEDDING_SDK.isEnabled();
+  const enableEmbeddingSdkSetting = useMergeSetting({
+    key: "enable-embedding-sdk",
+  });
+  const isEmbeddingSdkEnabled = Boolean(enableEmbeddingSdkSetting.value);
```

Comment:
Just curious, is the `Boolean(...)` cast because of how we store the value of settings as strings, or is this to cast nullish values to false?

Reply:
I don't mind the extra `Boolean(xxx)` if it makes our code even a tiny bit safer, we usually don't do that but what if we had a `if(isEmbeddingSdkEnabled===false)` somewhere?

Reply:
I don't think we usually compare booleans with `=== true` or `=== false` anyway and it should be rare.

My only reason to use `Boolean` is when TypeScript is complaining about type errors, since I usually don't cast it as null/undefined are already considered false.

That said, I don't mind leaving it here or removing it, but I kind of remember doing this in the first iteration when the value needs to be boolean.